kratos/mpi/utilities/gather_modelpart_utility.h

```diff
@@ -131,6 +132,70 @@ class KRATOS_API(KRATOS_MPI_CORE) GatherModelPartUtility
     template <class TDataType>
     void ScatterFromMaster(const Variable<TDataType>& rVariable);
 
+    /**
+     * @brief Function to gather entities from other partitions
+     * @details This function is intended to gather entities from other partitions. The map indicates the partitions to gather and the vector the entities to gather from each partition. In the current serial case it does nothing.
+     * @param rModelPart Model part to gather entities from other partitions
+     * @param rNodesToBring Nodes to gather from other partitions. It must be an ordered map to ensure the order of the ranks. This is a synchronous MPI implementation, and order matters until asynchronous is implemented.
+     * @param rElementsToBring Elements to gather from other partitions. It must be an ordered map to ensure the order of the ranks. This is a synchronous MPI implementation, and order matters until asynchronous is implemented.
+     * @param rConditionsToBring Conditions to gather from other partitions. It must be an ordered map to ensure the order of the ranks. This is a synchronous MPI implementation, and order matters until asynchronous is implemented.
+     * @param CallExecuteAfterBringingEntities Call Execute after gathering entities
+     * @param EchoLevel The verbosity level
+     */
+    static void GatherEntitiesFromOtherPartitions(
+        ModelPart& rModelPart,
+        const std::map<int, std::vector<std::size_t>>& rNodesToBring,
```

Comment:
can you please indicate what these maps contain? The Ids or the entities? Or their position in the vector of entities?